cluster-autoscaler/cloudprovider/gce/autoscaling_gce_client.go

```diff
@@ -259,7 +259,8 @@ func (client *autoscalingGceClientV1) DeleteInstances(migRef GceRef, instances [
 		req.Instances = append(req.Instances, GenerateInstanceUrl(i))
 	}
 	op, err := client.gceService.InstanceGroupManagers.DeleteInstances(migRef.Project, migRef.Zone, migRef.Name, &req).Do()
-	if err != nil {
+	wasConflictErr := op != nil && op.HttpErrorStatusCode == http.StatusConflict
```

Comment:
I'd create a new function to valuate the http status code for cleaner code, e.g.
```go
func isDeleteInstanceConflict(errorCode string) bool {
  return errorCode == http.StatusConflict
}

if !isDeleteInstanceConflict(op.HttpErrorStatusCode) && err != nil {
....
```